venues/ICML.cc/2017/MLAV/python/superuser-init.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+
+import openreview
+import sys, os
+import config
+import getpass
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "/Users/mandler/projects/openreview-scripts/utils"))
+import utils
+import templates
+
+args, parser, overwrite = utils.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+"""
+
+OPTIONAL SCRIPT ARGUMENTS
+
+	baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+ 	username - the email address of the logging in user
+	password - the user's password
+
+See openreview-scripts/utils for utility function and template details.
+
+"""
+
+if client.username.lower() != "openreview.net": raise(Exception('This script may only be run by the superuser'))
+
+# create groups for each directory level:  Conf.org, Conf.org/2017, Conf.org/2017/Workshop
+path_components = config.CONF.split('/')
+paths = ['/'.join(path_components[0:index+1]) for index, path in enumerate(path_components)]
+
+for p in paths:
+	if not client.exists(p) and p != config.CONF:
+		client.post_group(openreview.Group(
+			p,
+			readers = ['everyone'],
+			writers = [],
+			signatures = [],
+			signatories = [],
+			members = []
+		))
+		print "Posting group: ", p
+
+# CONF group needs a webpath - different configuation parameters
+if not client.exists(config.CONF):
+	conf_group = client.post_group(openreview.Group(config.CONF, **config.conf_params))
+	print "Posting group: ", config.CONF
+else:
+	print "Group %s already exists" % config.CONF
+	conf_group = client.get_group(config.CONF)
+
+## PAM check REVIEWERS correct?
+if not client.exists(config.REVIEWERS):
```

Comment:
why do you post the reviewer group here? shouldn't be done in the admin-init.py?  cc/ @mspector